dev/com.ibm.ws.repository.resolver/src/com/ibm/ws/repository/resolver/RepositoryResolutionException.java

```diff
@@ -329,4 +372,25 @@ private String getResourceName(RepositoryResource resource) {
         }
     }
 
+    /**
+     * @return the resolvedPlatforms
+     */
+    public Set<String> getResolvedPlatforms() {
+        return resolvedPlatforms;
+    }
+
+    /**
+     * @return the missingPlatforms
+     */
+    public Set<String> getMissingPlatforms() {
+        return missingPlatforms;
+    }
+
+    /**
+     * @return the missingBasePlatforms
+     */
+    public List<String> getMissingBasePlatforms() {
+        return missingBasePlatforms;
+    }
```

Comment:
Please add more documentation about what each of these fields means and what data it contains.

These fields look like they are used in other places, but here seems like the most logical place to document them.

Reply:
yes tried to add javadoc - one is simply a missspelling of a platform name - one is related to a base name like MP or jakarta realted to a versionless feature - if they can't be derived up front - its an error, and the reason the feature isn't resolved.